invenio_queues/config.py

```diff
@@ -8,6 +8,20 @@
 
 """Default configuration for QUEUES."""
 
+from kombu import Exchange
```

Comment:
```suggestion
```

I think it's an old import from testing

Reply:
No, this is used to define the queues' ``Exchange`` as here:

```python
app.config['QUEUES_DEFINITIONS'] = [
    {
        'name': 'notifications',
        'exchange': Exchange(
            'example',
            type='direct',
            delivery_mode='transient',  # in-memory queue
        ),
    },
]
```

Reply:
But it is not used in `config.py`. In the usage docs it has to be included (I think it is)

Reply:
Indeed was a leftover from testing